pkg/pxc/backup/volume.go

```diff
@@ -3,38 +3,77 @@ package backup
 import (
 	"fmt"
 
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	api "github.com/Percona-Lab/percona-xtradb-cluster-operator/pkg/apis/pxc/v1alpha1"
 )
 
-// PVC returns the list of PersistentVolumeClaims for the backups
-func PVC(cr *api.PerconaXtraDBBackup) (corev1.PersistentVolumeClaim, error) {
-	rvolStorage, err := resource.ParseQuantity(cr.Spec.Storage)
-	if err != nil {
-		return corev1.PersistentVolumeClaim{}, fmt.Errorf("wrong storage resources: %v", err)
-	}
+const volumeNamePostfix = "-backup-volume"
```

Comment:
maybe "volume" word in volume name is redundant.

Reply:
agree